authors/apps/articles/views/views.py

```diff
@@ -175,7 +175,6 @@ def get(self, request, slug):
 class TagListAPIView(generics.ListAPIView):
     """ Create a view that is used to fetch all the tags """
     pagination_class = None
```

Comment:
@kdaglas please set the permission class to allowany instead of none

Reply:
I have done that was pushing... thank you